sqlserver/datadog_checks/sqlserver/utils.py

```diff
@@ -25,6 +25,13 @@ def construct_use_statement(database):
     return 'use [{}]'.format(database)
 
 
+def get_stmt_is_proc(text):
+    if text:
+        t = text.upper()
+        return ("CREATE" in t) and (("PROC" or "PROCEDURE") in t)
```

Comment:
Why "PROC" ?

Reply:
Another improvement to reduce false positives:

```python
idx_create = t.index('CREATE')
idx_proc = t.index('PROCEDURE')

# ensure both keywords are found and CREATE occurs before PROCEDURE
idx_create >= 0 and idx_proc >= 0 and idx_create < idx_proc
```

Reply:
> Why "PROC" ?

Running `CREATE PROC` is also legit according to the official docs https://docs.microsoft.com/en-us/sql/t-sql/statements/create-procedure-transact-sql?view=sql-server-ver16